### ISSUE TITLE
release 0.2 before Workshop

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/pmda#59]
We need a stable release that pip-installs before the [workshop](https://workshop.mdanalysis.org)

- [x] tagged
- [x] GitHub release
- [x] PyPi release
- [ ] conda release

**Comment by kain88-de** [comment]
This needs to get hand-in-hand with a new MDAnalysis release. This library depends on features in the development branch.

**Comment by orbeckst** [comment]
cc @richardjgowers 
blocked on MDAnalysis/mdanalysis#1971

**Comment by orbeckst** [comment]
MDA 0.19.0 is out – need to get started on this release.

I'll wait for #47 and then get started.

**Comment by orbeckst** [comment]
Are we ready for a conda package, or should we keep it pip-only for 0.2.0?

**Comment by kain88-de** [comment]
There is nothing speaking against making a conda forge package. Since we
are pure python it straight forward.

**Comment by kain88-de** [comment]
the manifest.in should include the license file

**Comment by kain88-de** [comment]
conda-forge: https://github.com/conda-forge/staged-recipes/pull/6851

**Comment by orbeckst** [comment]
Conda package: #74

**Comment by orbeckst** [comment]
Waiting for PR #77 .

**Comment by orbeckst** [comment]
Conda release in progress https://github.com/conda-forge/pmda-feedstock/pull/2

**Comment by kain88-de** [comment]
There is a good conda not now. A PR should be created automatically about a
day after publishing on pypi

**Comment by orbeckst** [comment]
@kain88-de I did not understand your comment:
> There is a good conda not now.

Are you saying that it is not necessary to manually create a PR for the feedstock because it monitors PyPi? If this is the case, please comment on https://github.com/conda-forge/pmda-feedstock/pull/2 and we close that PR.

**Comment by orbeckst** [comment]
Ok, can you then head over to https://github.com/conda-forge/pmda-feedstock/pull/2 and do magic?

**Comment by kain88-de** [comment]
@orbeckst here is the bot https://github.com/regro/cf-scripts

**Comment by orbeckst** [comment]
Thanks!

**Comment by orbeckst** [comment]
Well, that should have been closed a long time ago because 0.2.0 was in https://anaconda.org/conda-forge/pmda by the workshop time.

I'll add a conda badge to our README...
